vendor/engines/secure_rooms/app/services/secure_rooms/access_handlers/occupancy_handler.rb

```diff
@@ -17,22 +17,30 @@ def initialize(event)
       end
 
       def process
+        Rails.logger.info("[SecureRooms] Entered SecureRooms::AccessHandlers::EventHandler#process")
```

Comment:
Do we want to add any IDs in there? I know we should be able to trace through the log by the request id, but should we include order detail ids and occupancy ids?